src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_RADIUS = 10;
+    private static final int QUANTITY_FIGURE = 5;
+    private static final int MIN_LENGTH_FIGURE = 1;
+    private static final int MAX_LENGTH_FIGURE = 100;
+    private static final String COLOR_CIRCLE = "WHITE";
+    private final Random random = new Random();
+    private final ColorSupplier supplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        String color = supplier.getRandomColor();
+        int indexFigure = random.nextInt(QUANTITY_FIGURE);
+
+        return switch (indexFigure) {
+            case 0 -> new Circle(
+                    color,
+                    random.nextInt(MIN_LENGTH_FIGURE, MAX_LENGTH_FIGURE + 1)
```

Comment:
create separate method `getRandomSide` for `random.nextInt(MIN_LENGTH_FIGURE, MAX_LENGTH_FIGURE + 1)` and use it for each type of figure